homeassistant/components/mqtt/__init__.py

```diff
@@ -39,7 +40,6 @@
 DATA_MQTT = 'mqtt'
 
 SERVICE_PUBLISH = 'publish'
-SIGNAL_MQTT_MESSAGE_RECEIVED = 'mqtt_message_received'
```

Comment:
As all callbacks are now handled by the `MQTT` class, the dispatcher calls are no longer required (*except for tests)

Reply:
The tests should test the actual code, so let's make sure it is also not needed in tests.

Reply:
Ok, I'll do that